install/antennas.csv

```diff
@@ -432,4 +432,5 @@ Trimble Navigation Ltd.,TRM115000.00,1551016811,WGTN,0.055,0,0,0,2020-02-14T01:1
 Trimble Navigation Ltd.,TRM115000.00,1551017047,METH,0.002,0,0,0,2019-09-20T00:11:38Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM115000.00,2612758627,RLTN,0.002,0,0,0,2022-07-10T23:59:08Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM115000.00,2612758639,NLS3,0,0,0,0,2022-08-24T04:31:08Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRM115000.00,2612758717,TGK1,0,0,0,0,2022-12-19T21:53:26Z,9999-01-01T00:00:00Z
```

Comment:
happy to the antenna height being 0, but I wonder if the type of mark that was used will have a calibration for the distance between geodetic mark and bottom of the antenna that we can use?

Reply:
If the antenna was twisted properly, I belive the difference in the heights of ARP and monument reference surface will be zero. But we need photos to find out this.